tests/storage/reservation.go

```diff
@@ -234,9 +234,11 @@ var _ = SIGDescribe("[Serial]SCSI persistent reservation", Serial, func() {
 				"there are NO registered reservation keys")).To(BeTrue())
 			Expect(checkResultCommand(vmi, "sg_persist -o -G  --param-sark=12345678 /dev/sda",
 				"Peripheral device type: disk")).To(BeTrue())
-			Eventually(checkResultCommand(vmi, "sg_persist -i -k /dev/sda",
-				"1 registered reservation key follows:\r\n    0x12345678\r\n"),
-				60*time.Second, 10*time.Second).Should(BeTrue())
+			Eventually(checkResultCommand,
+
```

Comment:
Is this empty line an artifact of some auto-formatting or a manual thing?

Reply:
Some kind of auto formatting. Will fix.